_includes/schema.html

```diff
@@ -0,0 +1,25 @@
+{% comment %}
+Includes:
+- schema, array of:
+    - key
+    - description
+    - type (optional)
+    - properties (optional)
+{% endcomment %}
+<ul>
+  {% for item in include.schema %}
+    <li>
+<div markdown="1">
```

Comment:
Presuming this is not indented b/c of the unfortunate quirk where things are escaped if they are indented more than four spaces and included in a markdown file

Reply:
yes :[